components/SSI/WalletInfo/index.tsx

```diff
@@ -201,7 +202,20 @@ function Component(props: InputType) {
                                 color: isLight ? '#000' : '#dbe4eb',
                             }}
                         >
-                            {loadingInfoBal ? <Spinner /> : infoBal} {currency}
+                            {loadingInfoBal ? (
+                                <Spinner />
+                            ) : infoBal === null &&
+                              currency.toLowerCase() === 'zil' ? (
+                                <div
+                                    onClick={() => updateShowZilpay(true)}
```

Comment:
the unlock zilpay button is throwing a toast only once but not opening the zilpay window

Reply:
This usually happen when you are alredy unlock and lock more than one time at one session